src/app-layout/__integ__/mobile.test.ts

```diff
@@ -50,7 +50,7 @@ describe.each(['classic', 'visual-refresh', 'visual-refresh-toolbar'] as const)(
       setupTest(async page => {
         await page.visit(pageName);
         const [documentWidth, windowWidth] = await page.getViewportWidths();
-        expect(documentWidth).toEqual(windowWidth);
+        expect(documentWidth).toBeLessThanOrEqual(windowWidth);
```

Comment:
The dev page has a page-level scrollbar which makes the inner document element be 15px smaller than the window.

Changing from `toEqual` to `toBeLessThanOrEqual` preserves the purpose of the test, which is to verify that the document is _not wider_ than the page (and thus overflowing horizontally).